src/index.js

```diff
@@ -0,0 +1,41 @@
+import './style.css';
+import refresh from './assets/refresh.png';
+import enter from './assets/enter.png';
+
+const refreshImg = document.getElementById('refreshImg');
+const enterImg = document.getElementById('enterImg');
+
+refreshImg.src = refresh;
+enterImg.src = enter;
+
+const list = document.getElementById('list');
+const tasks = [
+  {
+    index: 0,
+    description: 'Wash the dishes',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Complete To Do list project',
+    completed: false,
+  },
+];
+
+const addTask = (task) => {
+  list.innerHTML += `
+    <li class="task">
```

Comment:
- [ ] Kindly add the index to the list items also, this will ensure the lists is arranged numerically 😃.